### DISCUSSION TITLE
Max_acceleration

### DISCUSSION THREAD

**Discussion by Omikaa** [bitcraze/discussions#1265] [Q&A]
What is  the crazyflie 2.1 max_acceleration when it flies straight along Z in world corrdination?

**Comment by gemenerik** [comment]
Please split your questions into separate discussions. I'll answer your first question here.

The maximum acceleration will vary somewhat per drone, but checkout [this blogpost](https://www.bitcraze.io/2021/08/building-a-crazyflie-thrust-stand/) and [this documentation page](https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/functional-areas/pwm-to-thrust/) for some dummy values and explanation.

I'll answer your thrust command linearity question in your other thread; #1267
I do not understand your question about the "model's action to interp the command", please open a new discussion and clarify.

**Reply by Omikaa** [reply]
OK, thank you. I will check out
